- en: numpy.exceptions.TooHardError
  id: totrans-0
  prefs:
  - PREF_H1
  type: TYPE_NORMAL
  zh: numpy.exceptions.TooHardError
- en: 原文：[https://numpy.org/doc/1.26/reference/generated/numpy.exceptions.TooHardError.html](https://numpy.org/doc/1.26/reference/generated/numpy.exceptions.TooHardError.html)
  id: totrans-1
  prefs:
  - PREF_BQ
  type: TYPE_NORMAL
  zh: 原文：[https://numpy.org/doc/1.26/reference/generated/numpy.exceptions.TooHardError.html](https://numpy.org/doc/1.26/reference/generated/numpy.exceptions.TooHardError.html)
- en: '[PRE0]'
  id: totrans-2
  prefs: []
  type: TYPE_PRE
  zh: '[PRE0]'
- en: max_work was exceeded.
  id: totrans-3
  prefs: []
  type: TYPE_NORMAL
  zh: 超过了max_work上限。
- en: This is raised whenever the maximum number of candidate solutions to consider
    specified by the `max_work` parameter is exceeded. Assigning a finite number to
    max_work may have caused the operation to fail.
  id: totrans-4
  prefs: []
  type: TYPE_NORMAL
  zh: 当超过由`max_work`参数指定的需考虑的最大候选解数时，将引发此异常。 将有限数值分配给max_work可能会导致操作失败。
